geoips/plugins/modules/procflows/config_based.py

```diff
@@ -249,19 +249,40 @@ def get_bg_xarray(sect_xarrays, area_def, prod_plugin, resampled_read=False):
 
 
 def get_resampled_read(
-    config_dict, area_defs, area_def_id, sector_type, reader, fnames, variables
+    config_dict,
+    area_defs,
+    area_def_id,
+    sector_type,
+    reader_plugin,
+    reader_kwargs,
```

Comment:
These all just had "reader_kwargs" added, which caused them to go over the line lenght and split up on separate lines with black auto formatting.